### PR TITLE
New store for clothes

### PR DESCRIPTION
**Pull request by colindamelio**
Please review my code :)
Be thoughtful, constructive, and positive.

### CODE REVIEW

File: scripts.js [hunk 1]
```diff
@@ -1,51 +1,51 @@
...
-$(function(){
+const totalInventory = [
+  {
+    title: 'Bowie Tee',
+    url: 'images/bowie.jpg',
+    price: 19.99,
+    stock: 4,
+  },
+  {
+    title: 'Don\'t Know Tee',
```
Comment (beatricegeorgeslemire) [inline scripts.js:11]: Looking good! I'm spotting a little typo in 'don't'.

File: scripts.js [hunk 2]
```diff
@@ -1,51 +1,51 @@
...
+  },
+  {
+    title: 'Skeleton Hand Tee',
+    url: 'images/skeleton.jpg',
+    price: 30.00,
+    stock: 10,
+  },
+  {
+    title: 'HackerYou Hoodie',
+    price: 50.00,
```
Comment (beatricegeorgeslemire) [inline scripts.js:42]: It's looking like you didn't include an image url for this item, was that intentional or is there one you can add?

File: scripts.js [hunk 3]
```diff
@@ -1,51 +1,51 @@
...
+    stock: 4,
+  },
+]
+
+// document ready function allows page to load before running any of our scripts 
+$(function () {
```
Comment (beatricegeorgeslemire) [inline scripts.js]: Are we sure that this is a document ready function, I could be wrong, but I feel like there might be pieces missing.

File: scripts.js [hunk 4]
```diff
@@ -72,13 +72,41 @@ $(function(){
...
+    return item.url && item.stock;
+  });
+
+
+
+  // STEP TWO: write a function that displays inventory on the page in the correct pricing
+  // What kind of information (paramaters) does this funciton need to display all of our information?  
+
+  const displayItems = (inventory, currency) => {
+
```
Comment (beatricegeorgeslemire) [inline scripts.js]: This line might to be unnecessary? I love spaces out code, but maybe this one is extra :)

File: scripts.js [hunk 5]
```diff
@@ -72,13 +72,41 @@ $(function(){
...
 
+    inventory.forEach((item) => {
+
+      // the below syntax *creates* an html element as opposed to targeting one
```
Comment (beatricegeorgeslemire) [inline scripts.js]: Beautifully done!

**Comment by adammarostica** [comment]
Really great, clean work. You might consider adding a few comments to make it a little easier to read.

**Pull request closed without merging**